src/js/views/patients/patient/sidebar/sidebar_views.js

```diff
@@ -129,24 +135,29 @@ const sidebarWidgets = {
     className: 'widgets-value',
     template: hbs`{{ displayValue }}{{#unless displayValue}}{{{ defaultHtml }}}{{/unless}}`,
     templateContext() {
-      const fields = this.model.getFields();
-      const currentField = fields.find({ name: this.getOption('field_name') });
       const defaultHtml = this.getOption('default_html');
-
-      if (!currentField) return { defaultHtml };
-
-      const fieldValue = currentField.get('value');
-      const key = this.getOption('key');
-
-      const value = key ? propertyOf(fieldValue)(key) : fieldValue;
-      const phone = parsePhoneNumber(value, 'US');
+      const value = getFieldValue(this.model.getFields(), this.getOption('field_name'), this.getOption('key'));
+      const phone = parsePhoneNumber(value || '', 'US');
```

Comment:
so `parsePhoneNumber` needs a string or else it gets real sad

Reply:
ok well let's keep the `if (!value) return..` for instances like this one